src/main/java/hudson/tasks/junit/HistoryTestResultSummary.java

```diff
@@ -11,21 +13,19 @@ public class HistoryTestResultSummary {
     private final int skipCount;
     private final int passCount;
     private final String description;
-    private final hudson.tasks.test.TestResult resultInRun;
 
-    public HistoryTestResultSummary(Run<?, ?> run, hudson.tasks.test.TestResult resultInRun,
+    public HistoryTestResultSummary(Run<?, ?> run,
```

Comment:
this was the critical change, adding `resultInRun` broke the plugin and I can't see a way to retrieve it there. The API could be modified to maybe pass some more info down but it didn't seem necessary the only additional information that is missing now is the id of the test report which could be stored maybe if required